java/src/jmri/jmrit/beantable/AbstractTableAction.java

```diff
@@ -127,6 +122,23 @@ public BeanTableFrame<E> getFrame() {
         return f;
     }
 
+    /**
+     * Get the relevant data model for the current table.
+     * <p> This is overridden in the tabbed-table classes
+     * to return their own local data model.
+     * <p> Unlike {@link #getTableDataModel()}, this therefore
+     * doesn't attempt to (re)-create the model.
+     */
+    
+    public BeanTableDataModel<E> getDataModel() {
```

Comment:
I recommend to remove the empty line between the Javadoc and the method declaration. Otherwise it's easy to miss that the comment is a Javadoc for the method.

Reply:
Thanks for the careful review!  Done.